tests/test_marathon.py

```diff
@@ -109,16 +114,59 @@ def test_list_pod_returns_success_response_json():
     _assert_list_pod_returns_success_response_json(body_json=['a', 'b', 'c'])
 
 
-def test_list_pod_propagates_dcos_exception():
+def test_list_pod_propagates_rpc_dcos_exception():
     _assert_method_propagates_rpc_dcos_exception(
         lambda marathon_client: marathon_client.list_pod())
 
 
-def test_list_pod_propagates_json_parsing_exception():
-    _assert_method_propagates_json_parsing_exception(
+def test_list_pod_raises_dcos_exception_for_json_parse_errors():
+    _assert_method_raises_dcos_exception_for_json_parse_errors(
         lambda marathon_client: marathon_client.list_pod())
 
 
+def test_update_pod_executes_successfully():
+    _assert_update_pod_executes_successfully()
+    _assert_update_pod_executes_successfully(
+        pod_id='/foo bar/', path='v2/pods/foo%20bar')
+    _assert_update_pod_executes_successfully(
+        force=True, params={'force': 'true'})
+    _assert_update_pod_executes_successfully(
+        pod_json={'something': 'different'})
+    _assert_update_pod_executes_successfully(
+        response_body={'deploymentId': 'an-arbitrary-value'},
+        expected_return='an-arbitrary-value')
```

Comment:
Contrast these test cases with the ones in `unit/test_marathon_pod.py` for the `test_pod_update_invoked_successfully()` method. Here, I used default argument values for the "common" inputs, so the only values shown are the ones that differ. There, I always give all argument values, even if they are repetitive. Not sure which style is preferable; I can see pros and cons for each.